docs/bookstore/page-5/deploy-database-service.md

```diff
@@ -0,0 +1,88 @@
+# **Deploy the Database Service**
+
+![image1](images/image1.png)
+
+## **What Knative features will we learn about?**
+
+- What is a good usecase to use Knative Service
+
+## **What does the final deliverable look like?**
+
+- Running a PostgreSQL StatefulSet that contains the table and some sample data
+
+## **Concept Learning**
+![image4](images/image4.png)
+
+**Knative Services** are a powerful feature within the Knative ecosystem, designed to handle a wide range of use cases, especially in modern cloud-native applications, it can be controlled by Knative Serving, and achieve scale to 0. Here's an expanded explanation of when and why you should consider using Knative Services:
+
+???+ danger "Stateless Workloads"
+    - **Definition:** Stateless applications do not store any data locally between requests. Each request is independent and does not rely on any previous interaction.
+    - **Use Case:** Examples include web servers, APIs, and microservices where the state is managed externally, such as in a database or a cache.
+    - **Benefits:** Simplifies scaling and failover because any instance can handle any request without requiring session persistence.
+
+???+ example "Event-Driven Workloads"
+    - **Definition:** Event-driven architectures respond to events or Triggers, such as HTTP requests, messages in a queue, or changes in a database.
+    - **Use Case:** Use Knative Services to deploy functions that react to events, such as processing incoming data, triggering workflows, or integrating with third-party APIs.
+    - **Benefits:** Efficient resource utilization, as services can scale down to zero when not handling events, reducing costs and improving performance.
+
+![image6](images/image6.png)
+
+Try to ask in the Knative Slack community [#knative](https://cloud-native.slack.com/archives/C04LGHDR9K7){:target="_blank"} whether it is the best use case or not.
+
+## **Implementation**
+
+### **Step 1: apply all the config yaml files**
+
+![image9](images/image9.png)
+
+In this section, we will just be simply running a PostgreSQL service. We have all config files ready. Simply run the following command to apply all yamls at once.
+
+```sh
+kubectl apply -f db-service
```

Comment:
I guess I'd list each file, and have one / two sentences on what that is .

This DB is option, but just applying a bag of yaml is not always helpful :-)

Reply:
I'm on the opposite opinion as it's not the point of the tutorial to explain how to deploy a database on Kubernetes

Reply:
We don't explain it in here, but we have the detailed explanation in the [db-service README.md ](https://github.com/knative/docs/blob/main/code-samples/eventing/bookstore-sample-app/solution/db-service/README.md)